server/get-browsers.js

```diff
@@ -0,0 +1,106 @@
+import browserslist from 'browserslist'
+import { readFileSync } from 'fs'
+import { URL } from 'url'
+import { agents as caniuseAgents, region as caniuseRegion } from 'caniuse-lite'
+
+let { version: bv } = importJSON('./node_modules/browserslist/package.json')
+let { version: cv } = importJSON('./node_modules/caniuse-lite/package.json')
+let wikipediaLinks = importJSON('../browsers-data/wikipedia-links.json')
```

Comment:
Should we do it on client side?

My idea that this API will be used not only by our client.

Reply:
> My idea that this API will be used not only by our client.

And I thought that we are doing the backend only the task of the site browsersl.ist)

> Should we do it on client side?
 
Let's leave JSON with links and images from Wikipedia only on the client side